scripts/run-django-dev.sh

```diff
@@ -6,9 +6,7 @@ python3 manage.py collectstatic --noinput --clear
 python3 manage.py migrate --noinput
 RUN_DATA_MIGRATIONS=true python3 manage.py migrate --noinput
 
-if [[ $NODE_ENV == "development" ]]; then
-	# load required fixtures on development by default
-	python3 manage.py loaddata platforms schools departments offered_by
-fi
+# load required fixtures on development by default
+python3 manage.py loaddata platforms schools departments offered_by
```

Comment:
The block above was never running. We don't expose `NODE_ENV` to the web container, nor have we for a while now.

Since `run-django-dev.sh` is ...for dev ... it seems OK to just always run this line.

That said, also happy to remove this (and restore it to the README).

Reply:
This is fine, I think. I can't imagine a scenario where you would need to preserve manual edits to any of this data in development, but it might be best to add a quick note to the "Loading fixture files" section of the readme. There's a note about how it's automatically done for you in development, so maybe just elaborate upon that and say that in development data is loaded from fixtures *every time*, which is what this would do as far as I understand.